skbio/diversity/alpha/_base.py

```diff
@@ -819,10 +807,11 @@ def observed_otus(counts):
     `observed_otus` is an alias for `sobs`.
 
     """
+    warn("observed_otus is deprecated as of 0.6.0.", DeprecationWarning)
```

Comment:
Note, this is going to cause a break with [`q2-diversity-lib`](https://github.com/qiime2/q2-diversity-lib/blob/dev/q2_diversity_lib/alpha.py#L74-L75)

Reply:
@wasade Thanks! Do you mean that `q2-diveristy-lib` will print lots of warnings which are unfavorable, or that it will just fail to run?

Reply:
Sorry once this method is actually removed, merged and released, then q2 would raise an exception, right?

Reply:
Hi @wasade The current PR won't remove `observed_otus`. I don't think it will be removed soon (at least not in a few future q2 releases). Therefore, once this PR is merged, I don't think q2 will raise an exception (I guess a warning is not an exception).

Reply:
I know, the comment is in regards to when it is removed